deploy/roles/sqitch/tasks/main.yml

```diff
@@ -40,7 +40,7 @@
   register: timezone
   changed_when: '"is up to date" not in timezone.stdout'
   environment:
-    PATH: "{{ postgres_install_dir|default('/usr/pgsql-9.4') }}/bin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin"
+    PATH: "{{ postgres_install_dir|default('/usr/pgsql') }}/bin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin"
```

Comment:
Since the above change means that the symlink always points to "postgres_install_dir", it is probably fine to simplify this and just use '/usr/pgsql' only (below as well)